dart/lib/src/http_client/sentry_http_client.dart

```diff
@@ -83,7 +83,6 @@ class SentryHttpClient extends BaseClient {
     MaxRequestBodySize maxRequestBodySize = MaxRequestBodySize.never,
     List<SentryStatusCode> failedRequestStatusCodes = const [],
     bool captureFailedRequests = false,
```

Comment:
You can move this flag to `SentryOptions`, since `captureFailedRequests` is used by 2 integrations, `SentryHttpClient` and `SentryDioExtension`